examples/complete/main.tf

```diff
@@ -3,17 +3,9 @@ module "repo_config" {
 
   repos = [
     {
-      id       = "github.com/getindata/foo"
```

Comment:
This seemed like a better example then your version

Reply:
ok , fixed